float8_experimental/float8_tensor.py

```diff
@@ -325,6 +325,8 @@ def allowed_subclasses(type):
 
         if func in FLOAT8_OPS_TABLE:
             return FLOAT8_OPS_TABLE[func](func, args, kwargs)
+        else:
+            return func.decompose(*args, *kwargs)
```

Comment:
I'm not sure this is what you want, it directly invokes the CompositeImplicitAutograd implementation

Reply:
I think this is exactly what they want :D 
What happens if there is no CompositeImplicitAutograd impl?

Reply:
it returns NotImplemented

Reply:
yeah I figure that it we can noop ( execept not implemented) slide our way down by trying to decompose the op 
- Either it wont be decomposable, and return not implemented, cool same place as after this
- or it is decomposable:
	- Run supported op, Great!
	- Not supported and ultimately end back up with a not implemented